File: JS/G_Factorial.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/).map(Number);

function factorial(num) {
    let ans = 1;
    for (let i = 1; i <= num; i++) ans *= i;
    return ans;
}

let idx = 0;
let test = input[idx++];
while (test--) {
    let n = input[idx++];
    console.log(factorial(n));
}

File: C/G_Factorial.c
#include <stdio.h>

int main() 
{
    int test;
    scanf("%d", &test);
    while (test--)
    {
        int num;
        scanf("%d", &num);
        long long ans = 1;
        for (int i = 1; i <= num; i++) ans *= i;
        printf("%lld\n", ans);
    }

    return 0;
}